e2e-tests/insurance/cypress/e2e/journeys/declarations/modern-slavery/save-and-back.spec.js

```diff
@@ -0,0 +1,208 @@
+
+      cy.assertAllSectionsUrl(referenceNumber);
+
+      cy.checkTaskDeclarationsAndSubmitStatusIsInProgress();
+    });
+
+    describe('when going back to the page', () => {
+      it('should have the submitted value', () => {
+        cy.navigateToUrl(url);
+
+        cy.completeModernSlaveryForm({
+          willAdhereToAllRequirements: null,
+          hasNoOffensesOrInvestigations: true,
+          isNotAwareOfExistingSlavery: null,
+        });
+
+        cy.assertYesRadioOptionIsChecked(1);
+      });
+    });
+  });
+
+  describe('when all fields are submitted with `yes` radios', () => {
+    // TODO: EMS-4059
+    it.skip('should retain the status of task `declarations and submit` as `in progress`', () => {
+      cy.navigateToUrl(url);
+
+      cy.completeModernSlaveryForm({
+        willAdhereToAllRequirements: true,
+        hasNoOffensesOrInvestigations: true,
+        isNotAwareOfExistingSlavery: true,
+      });
+
+      cy.clickSaveAndBackButton();
+
+      cy.navigateToAllSectionsUrl(referenceNumber);
+
+      cy.checkTaskDeclarationsAndSubmitStatusIsInProgress();
+    });
+
+    describe('when going back to the page', () => {
+      it('should have the submitted values', () => {
+        cy.navigateToAllSectionsUrl(referenceNumber);
+
+        // TODO: EMS-4041
+        // go through the first 4 declaration forms.
+        // cy.clickSubmitButtonMultipleTimes({ count: 4 });
+        cy.navigateToUrl(url);
+
+        cy.completeModernSlaveryForm({
+          willAdhereToAllRequirements: true,
+          hasNoOffensesOrInvestigations: true,
+          isNotAwareOfExistingSlavery: true,
+        });
+
+        cy.assertYesRadioOptionIsChecked(0);
+        cy.assertYesRadioOptionIsChecked(1);
+        cy.assertYesRadioOptionIsChecked(2);
+      });
+    });
+  });
+
+  describe('when all fields are submitted with `no` radios', () => {
+    // TODO: EMS-4059
+    it.skip('should retain the status of task `declarations and submit` as `in progress`', () => {
+      cy.navigateToUrl(url);
+
+      cy.completeModernSlaveryForm({
+        willAdhereToAllRequirements: false,
+        hasNoOffensesOrInvestigations: false,
+        isNotAwareOfExistingSlavery: false,
+        submitConditionalFields: true,
+      });
+
+      cy.clickSaveAndBackButton();
+
+      cy.navigateToAllSectionsUrl(referenceNumber);
+
+      cy.checkTaskDeclarationsAndSubmitStatusIsInProgress();
+    });
+
+    describe('when going back to the page', () => {
+      beforeEach(() => {
+        cy.navigateToUrl(url);
+      });
+
+      it('should have the submitted radio values', () => {
+        cy.completeModernSlaveryForm({
+          willAdhereToAllRequirements: false,
+          hasNoOffensesOrInvestigations: false,
+          isNotAwareOfExistingSlavery: false,
+          submitConditionalFields: true,
+        });
+
+        cy.clickSaveAndBackButton();
+
+        cy.navigateToAllSectionsUrl(referenceNumber);
+
+        cy.navigateToUrl(url);
```

Comment:
could this be in the beforeEach

Reply:
Thing is, this will all change and not require a `beforeEach` when EMS-4059 is complete, the structure of this test is temporary until EMS-4059 is in place. Would prefer to keep as is, otherwise we'll be spending time moving things around that will only change anyway.

Also note this is going into `main-declarations-modern-slavery`, not `main`.

EMS-4059 will align this with all other "Save and back" tests